recipe/meta.yaml

```diff
@@ -12,15 +12,17 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  skip: True  # [py<36]
 
 requirements:
   host:
     - pip
+    - setuptools
+    - wheel
     - python
   run:
-    - python >=3.6
+    - python
     - smartypants >=1.8.3
```

Comment:
Seems like `jinja2` can be in `run_constrained`, see https://github.com/mintchaos/typogrify/tree/2.0.7#requirements and https://github.com/mintchaos/typogrify/blob/2.0.7/typogrify/templatetags/jinja_filters.py

Reply:
true ... same as django ... so I am not sure if we want to mention one, but not the other ... so I think both are  option ones, so not necessarily be present